build/.moduleignore

```diff
@@ -138,6 +138,15 @@ vsda/SECURITY.md
 vsda/targets
 !vsda/build/Release/vsda.node
 
+# {{SQL CARBON EDIT}} Remove vscode-encrypt after 1.47 release
```

Comment:
I would recommend we keep it longer than 1.47 (until majority users have migrated their secrets), since based on our telemetry most of our users don't update instantly with new release and we may land up into same issues in future..

Reply:
Sure, I'll let you all make that call

Reply:
Either way, can you open an issue and add it to next milestone and assign it to me? I will follow up on telemetry data and keep it on track to remove it sometime in future. And you can update comment to reflect a future release instead of 1.47.